draft-ietf-secevent-subject-identifiers.md

```diff
@@ -461,6 +480,13 @@ Defining Document(s)
 * Change Controller: IETF
 * Defining Document(s): {{sub-ids}} of this document.
 
+#### Decentralized Identifier Format
+
+* Type Name: `did`
+* Type Description: Subject identifier based on a Decentralized Identifier (DID) URL.
+* Change Controller: IETF
```

Comment:
Hmm. Is this right? Should this be W3C instead of IETF? Is this enough of a reason to pull this out of this doc and register it in a doc published by W3C?

Reply:
As far as I know this is correct, but we can bring it up in IESG review if needed. @wseltzer do you have input here, for who's the change controller for something registered in an IETF document that refers to a W3C document?